src/main/java/io/codelex/classesandobjects/practice/Exercise2/LitersPerKilometer.java

```diff
@@ -0,0 +1,26 @@
+package io.codelex.classesandobjects.practice.Exercise2;
+
+import java.util.Scanner;
+
+class LitersPerKilometer {
+    /* public static void main( String[] args )  {
```

Comment:
Kāpēc šis aizkomentēts?

Reply:
Es tajā brīdī vēl nesapratu (vai neiedomājos), ka vienai klasei var būt divi dažādi konstruktori un sekoju uzdevuma aprakstam, kur bija teikts:
1. Constructors:

**Car( double startOdo, double endingOdo, double liters )**

un nedaudz vēlāk:

**Change the constructor for the Car class so that it has only one parameter, the first reading of the odometer.**

Secīgi, LitersPerKilometers es aizkomentēju pēc nevajadzības, jo tas izmantoja veco konstruktoru un main klasē bija jauna un labāka metode. Es sapratu, ka tas bija vajadzīgs tikai lai pārbaudītu vai Car objekti tiek pareizi izveidoti.

Tagad pielikšu klāt vēl vienu konstruktoru Car klasē lai LitersPerKilometers metode arī strādātu.